src/routes/transactions/entities/confirmation-view/confirmation-view.entity.ts

```diff
@@ -92,7 +92,7 @@ export class CowSwapConfirmationView implements Baseline, OrderInfo {
     type: String,
     description: 'The URL to the explorer page of the order',
   })
-  explorerUrl: URL;
+  explorerUrl: string;
```

Comment:
This change was necessary for tests to pass. This is already returned as a string by the controller but within the mapper it was a `URL` instance.